dist/api_data.py

```diff
@@ -364,7 +364,17 @@ def __cmp__(self, other):
     Config('eviction_trigger', '95', r'''
         trigger eviction when the cache is using this much memory, as a
         percentage of the total cache size''', min=10, max=99),
-    Config('lsm_manager', '', r'''
+    Config('file_close', '', r'''
+        control how aggressively file handles are closed''',
```

Comment:
I'd prefer a name like `file_handle_manager` or `file_manager` or something more generic.  We may someday want to add something not related to close such as maximum file handles allowed or whatever.  The description could then say "control how file handles are managed".  I think it still works well even if both of the subcategory types currently relate to close.